src/renderer/widgets/SendAssetModal/model/send-asset.ts

```diff
@@ -15,17 +52,175 @@ const getConfigFx = attach({ effect: xcmModel.effects.getConfigFx });
 const saveConfigFx = attach({ effect: xcmModel.effects.saveConfigFx });
 const fetchConfigFx = attach({ effect: xcmModel.effects.fetchConfigFx });
 
+const getParaIdFx = createEffect(async (api: ApiPromise): Promise<number | null> => {
+  try {
+    return getParachainId(api);
+  } catch (e) {
+    return null;
+  }
+});
+
 forward({
   from: xcmConfigRequested,
   to: [getConfigFx, fetchConfigFx],
 });
 
+sample({
+  source: { asset: assetGuardModel.$asset, chain: assetGuardModel.$chain, config: $finalConfig },
+  filter: (props: {
+    asset: Asset | null;
```

Comment:
Using props via `Gate` will make sure that `chain` and `asset` are always defined
Otherwise we won't event be allowed to open `SendAssetModal`